### PR TITLE
[RecoveryServices] Sql restore accepting filestream DBs

### PR DESCRIPTION
**Pull request by siddharth7**
## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs [hunk 1]
```diff
@@ -239,7 +239,15 @@ public string GetTargetPath(string path, string name, string type, List<SQLDataD
             List<string> pathParts = new List<string>(path.Split(new string[] { "\\" }, StringSplitOptions.None));
             int len = pathParts.Count();
             List<string> fileNameParts = new List<string>(pathParts[len - 1].Split(new string[] { "." }, StringSplitOptions.None));
-            string newFileName = name + "_" + offset.ToString() + "." + fileNameParts[1];
+            string newFileName = "";
+            if (fileNameParts.Count == 2)
```
Comment (isra-fel) [inline src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs]: Can you write a unit test for this?

Comment (siddharth7) [reply src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs]: @isra-fel The present test i.e. `TestAzureVmWorkloadFullRestore` covers this scenario.

Comment (isra-fel) [reply src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs]: Does it cover the new logic you added? Where `filenameparts.count != 2`

Comment (siddharth7) [reply src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs]: Added

**Pull request merged**